developers/weaviate/current/configuration/authentication.md

```diff
@@ -161,9 +178,15 @@ We outline the steps below for both methods of obtaining a token.
 5. Go to the `auth_url` in your browser, and log in if prompted. If successful, the token issuer will redirect the browser to the `redirect_url`, with additional parameters that include an `id_token` parameter. 
 6. Parse the `id_token` parameter value. This is your Bearer token.
 
-#### Code example
+### OIDC support for Weaviate clients
+
+The latest versions (from mid-December 2022 and onwards) of Python, JavaScript, Go and Java Weaviate clients support OIDC authentication. If Weaviate is set up to use the `client credentials grant` flow as or `resource owner password flow`, the respective Weaviate client can instantiate a connection to Weaviate that incorporates the authentication flow.
+
+Please refer to the relevant documentation for each client for code examples.
```

Comment:
I'd move this further up, near "how to use". We could also link to "Client libraries" here?